examples/transfer_files/transfer_files.py

```diff
@@ -169,7 +167,7 @@ def main(
 
     log.info("Connecting to the data transfer service client..")
     ##############################
-    # Create a ``client`` instance
+    # Create a client instance
```

Comment:
This ReStructuredText formatting within the script also looks odd in the rendered documentation and perhaps should be changed.

Reply:
Please look at the PR mentioned in previous comment addressing examples